ui/src/app/subnets/views/FabricDetails/FabricDetails.tsx

```diff
@@ -23,6 +25,7 @@ const FabricDetails = (): JSX.Element => {
   );
   const fabricsLoading = useSelector(fabricSelectors.loading);
   const isValidID = isId(id);
+  useWindowTitle(`${fabric?.name || "Fabric"} details`);
```

Comment:
From what I've seeen SectionHeader is used as a main page wrapper (and its title prop is rendered as an h1).

What about we add the `useWindowTitle` hook to the SectionHeader component instead? Seems like the H1 and page title should match anyway + this will reduce duplication.

Reply:
e.g. `<title />` `<h1 />` should probably be the same string below:

![image](https://user-images.githubusercontent.com/7452681/150798986-a9d17d43-0db5-46ac-b468-dee2ae6f6eec.png)

Reply:
That's not a bad idea, I think the consistency between title and h1 would be great. I'll leave it as-is for now, just because this is how we've done it elsewhere, but I've added an issue so we can make the change to all the components simultaneously (maybe by setting the window title in `SectionHeader` like you suggested)

Reply:
Awesome, thanks!